lib/board.rb

```diff
@@ -43,6 +43,26 @@ def available_squares
     squares.select { |square| square.is_a? Integer }
   end
 
+  def copy_board
```

Comment:
@Kotauror, I agree it would make the method more universal. However, I'm unsure where other squares (that are not the squares of the instance of board) would be passed in